documentation/writing_plans.md

```diff
@@ -196,11 +203,19 @@ plan mymodule::myplan {
 }
 ```
 
-## Success and failure in plans
+## Indicating success and failure in plans
 
-Indicators that a plan has run successfully or failed.
+Any plan that completes execution without an error is considered successful. The
+`bolt` command exits `0` and any calling plans continue execution. If any calls
+to `run_` functions fail **without** `_catch_errors`, the plan halts execution
+and is considered a failure. Any calling plans also halt until a `run_plan` call
+with `_catch_errors` or a `catch_errors` block is reached. If one isn't reached,
+the `bolt` command exits `2`. 
```

Comment:
I'm not sure where this exits `2`, but if I have a plan fail the exit code is `1`? I think we can probably just remove this altogether, since it might fail with different exit codes in other scenarios.